docs/howto/write-integration-tests-for-a-charm.md

```diff
@@ -132,7 +132,7 @@ For `oci-images` you can reference an image registry.
 
 ### Test a relation
 
-To test an integration between two applications, you can just integrate them through
+To test a relation between two applications, you can just integrate them through
```

Comment:
Drive-by: make it less just:

```suggestion
To test a relation between two applications, you can integrate them through
```

Reply:
Counter, to be less just and less passive:

```suggestion
To test a relation between two applications, integrate them through
```